cmd/gcp-controller-manager/node_csr_approver.go

```diff
@@ -1007,6 +1019,63 @@ func deleteAllPodsBoundToNode(ctx *controllerContext, nodeName string) error {
 	return utilerrors.NewAggregate(errs)
 }
 
+func markFailedAndDeletePodWithCondition(ctx *controllerContext, pod *v1.Pod, condition *corev1apply.PodConditionApplyConfiguration) error {
+	fieldManager := "GCPControllerManager"
+
+	// Copied directly from pod gc controller (https://github.com/kubernetes/kubernetes/blob/17bf864c1fc20e4badc9a6e0659c5190310462e1/pkg/controller/podgc/gc_controller.go)
+	if utilfeature.DefaultFeatureGate.Enabled(features.PodDisruptionConditions) {
+
+		// Extact the pod status as PodGC may or may not own the pod phase, if
```

Comment:
Update the comment to use `GCPControllerManager`.

Reply:
Done